src/MicrosoftTeams.schema.json

```diff
@@ -43,79 +43,136 @@
                             },
-                            "full": {
-                                "type": "string",
-                                "description": "The full description of the extension. Maximum length is 256 characters.",
-                                "maxLength": 256
-                            }
-                        },
-                        "required": [
-                            "short",
-                            "full"
-                        ]
-                    },
-                    "icons": {
-                        "type": "object",
-                        "description": "Each icon image file must be a transparent PNG, with a white or light-colored background.",
-                        "properties": {
-                            "44": {
+                            "people": {
                                 "type": "string",
-                                "description": "An icon for the extension sized to 44x44."
+                                "description": "A value of 'people' indicates the tab offers an experience in the context of a chat between one or more people"
                             },
-                            "88": {
+                            "personal": {
                                 "type": "string",
-                                "description": "An icon for the extension sized to 88x88."
+                                "description": "A value of 'personal' indicates the tab offers an experience scoped to an individual user alone"
                             }
-                        },
-                        "required": [
-                            "44",
-                            "88"
-                        ]
+                        }
+                    }
+                },
+                "required": [
+                    "configUrl",
+                    "scope"
+                ]
+            }
+        },
+        "staticTabs": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "entityId": {
+                        "type": "string",
+                        "description": "A unique identifier for the entity which the tab displays. The id must use reverse domain name notation.",
+                        "maxLength": 64
                     },
-                    "accentColor": {
+                    "name": {
                         "type": "string",
-                        "description": "A color to use in conjunction with the extension's icons. The value must be a valid HTML color code starting with '#', for example `#4464ee`."
+                        "description": "The display name of the tab.",
+                        "maxLength": 16
                     },
-                    "configUrl": {
+                    "canvasUrl": {
```

Comment:
>canvasUrl [](start = 21, length = 9)

We actually call this "contentUrl" in the SDK when configuring a tab (see: [https://github.com/OfficeDev/microsoft-teams-library-js/blob/master/src/MicrosoftTeams.ts#L320](https://github.com/OfficeDev/microsoft-teams-library-js/blob/master/src/MicrosoftTeams.ts#L320)). Should we use a consistent term here?